src/core/validators/validators.ts

```diff
@@ -7,15 +7,15 @@
  * Returns null if no validation error
  */
 export type ValidatorFn = (
-  value: string
+  value?: string | [],
 ) => Promise<{ [id: string]: boolean }> | Promise<null>;
 
 /**
  * Checks if a mandatory value isn't null/undefined/empty , returns {required:true} if error, return null if it's not null/undefined/empty
  * @param value Value to test
  */
 const Required: ValidatorFn = (value) => {
-  if (!value || value?.trim?.() === '') {
+  if (!value?.length || (typeof value==='string' && value?.trim?.() === '')) {
```

Comment:
Why do you need to test the type of value? IMO, it's not needed because the strict operator ('===' ) already checks the type, but maybe I missed something :).

Reply:
Any reason for this change? this validator is already working as we expect I think ? maybe you found a bug somewhere?

Reply:
Let's keep it because it solves a Typescript error message. We should type all untyped variables in a next major release. ;)